src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -487,6 +487,10 @@
     text: az acr repository update -n MyRegistry --image hello-world:latest --write-enabled false
   - name: Update the attributes of the image referenced by digest 'hello-world@sha256:abc123' to disable write operation.
     text: az acr repository update -n MyRegistry --image hello-world@sha256:abc123 --write-enabled false
+  - name: Update the attributes of the Repository to disable Teleport feature.
+    text: az acr repository update -n MyRegistry --teleport-enabled false
```

Comment:
Can you add the repository parameter to make the sample command complete?

Reply:
oops good catch :D